README.md

```diff
@@ -2,13 +2,47 @@
 
 Kubernetes stack cookbook to work with Kubernetes
 
+## Requirements
 
-## How to use
+- Chef 12.5.x or higher. Chef 11 is NOT SUPPORTED.
+
+## Platform Support
```

Comment:
Platform support (the 2nd time and no changes after my 1st comment, this is bad)